packages/legacy/core/App/layout/wrapper.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { PropsWithChildren } from 'react'
+import { StyleSheet, View, ViewStyle } from 'react-native'
+import { Edge, SafeAreaView } from 'react-native-safe-area-context'
+
+import { TOKENS, useServices } from '../container-api'
+import { Screens } from 'types/navigators'
+
+export interface LayoutProps {
+  safeArea?: boolean
+  customEdges?: Edge[]
+  style?: ViewStyle
+  Header?: React.FC | undefined
+}
+
+const defaultStyles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+})
+
+type Props = {
+  screen: Screens
+}
+
+const ScreenLayout: React.FC<PropsWithChildren<Props>> = ({ children, screen }) => {
+  //safeArea, customEdges, style, header
+  const [screenLayoutOptions] = useServices([TOKENS.OBJECT_LAYOUT_CONFIG])
+  const screenProps = screenLayoutOptions[screen]
+  const { safeArea, customEdges, style, Header } = screenProps || {
+    safeArea: false,
+    customEdges: ['top', 'left', 'right', 'bottom'],
+    style: {},
+    header: undefined,
+  }
```

Comment:
![image](https://github.com/user-attachments/assets/00c25c60-7e82-49ab-9ddc-8c82c9aa0523)

Should be:
```const { safeArea, customEdges, style, header: Header }```

Reply:
Good catch, will fix it